test/views/declaration/DeclarationChoiceViewSpec.scala

```diff
@@ -17,26 +17,18 @@
 package views.declaration
 
 import base.Injector
```

Comment:
I don't think there should be any changes to this test either.  Declaration Choice page as already been GDS - its not part of this ticket